roles/thingsboard-enhancer/templates/thingsboard-to-parkapi-rt.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
```

Comment:
If both commands are run every 2 minutes we can use the same service file and timer, if you're ok with that.